packages/atlas/lib/src/padding.dart

```diff
@@ -0,0 +1,43 @@
+class Padding {
```

Comment:
Instead of creating a new class for padding, you can use the existing EdgeInsets from the flutter framework